Renumbering.py

```diff
@@ -93,14 +93,16 @@
             elif k >= 25 and k < 25 + natoms:
                 if len(L) == 8:
                     i = i + 1
-                    missing_linkers.write(L[1] + str(atoms_index_new[i]) + "   " + L[1] + "   "
-                              + L[2] + "   " + L[3] + "   "
-                              + L[4] + "   " + L[5] + "   " + L[6] + "   " + L[7] + "\n")
+                    missing_linkers.write(
+                    L[1] + str(atoms_index_new[i]) + "   " + L[1]
+                    + "   " + L[2] + "   " + L[3] + "   " + L[4]
+                    + "   " + L[5] + "   " + L[6] + "   " + L[7] + "\n")
```

Comment:
So PEP gives you some flexibility here, and I would definitely indent this. The way you had this before wasn't bad,  though it looks like you have some extra white space. When it all left justifies to the same line, it is impossible to tell which lines are part of the original line.

Also, this code can be much simpler using join, e.g.: `" ".join(["a", "b", "c"])` => 'a b c'